.golangci.yml

```diff
@@ -5,6 +5,8 @@ run:
   timeout: 2m
   skip-dirs:
     - vendor
+  build-tags:
```

Comment:
I will leave a build tag since we will have a lot of code duplication without this tag.
I think is no big deal to add a build tag.
Also, I can create PR for the essential library to refactor these build tags.